src/main/java/com/mallowigi/search/parsers/HexColorParser.kt

```diff
@@ -47,8 +47,9 @@ class HexColorParser internal constructor(prefix: String) : ColorParser {
    */
   private fun parseHex(text: String): Color =
     when (text.length) {
-      3 + offset -> ColorUtils.getShortRGB(text.substring(offset))
-      8 + offset -> ColorUtils.getRGBA(text.substring(offset))
-      else -> ColorUtils.getRGB(text.substring(offset))
+      3 + offset -> ColorUtils.getShortRGB(text.substring(offset)) // RGB
+      8 + offset -> ColorUtils.getRGBA(text.substring(offset)) // RRGGBBAA
+      6 + offset -> ColorUtils.getRGB(text.substring(offset)) // RRGGBB
+      else -> throw NullPointerException()
```

Comment:
I'm not sure we should throw an exception. Did you check this ?

Reply:
The function is called from `getColor`, which catches every `RuntimeException` and returns `null` in such case.

However returning null is easy, the interface of `ColorParser` contains `parseColor` returning nullable, therefore I have changed it that way.